### PR TITLE
Add portfolio mobile

### PR DESCRIPTION
**Pull request by bimbolabuari**
# Portfolio Microverse Project 🥇 

 > This PR adds a project about building a portfolio with  HTML and CSS that demonstrates the knowledge of Semantic HTML, flexbox, background properties, and mobile-first responsiveness.

## Built With 😉  

- Semantic HTML & CSS
- Linters (Stylelint, Web hint, Lighthouse)

## Live Preview 🚀 

[Live Preview Link](https://deploy-preview-1--bimbola-mv-portfolio.netlify.app/)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -110,6 +107,296 @@
...
+  font-size: 30px;
+  text-align: center;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 52px;
+  margin-bottom: 30px;
+}
+
+.works {
+  flex-direction: column;
```
Comment (Davidosky007) [inline styles.css:222]: The main container of the projects should be display:grid as stated in the project requirement

File: index.html [hunk 2]
```diff
@@ -68,5 +68,166 @@ <h3 class="title">I'am a Software Developer</h3>
              </li>
         </ul>
     </section>
+    <section class="work">
+      <h2 class="work-header">My recent Works</h2>
+      <div class="works flex">
+        <img class="img-works" src="./images/main-project-setup.jpg" alt="project" />
```
Comment (Davidosky007) [inline index.html]: I suggest you use background images here as stated in the project requirements: 
(decorations are always background images, not `img` tags)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @bimbolabuari, 

Good job so far! 👍 your project looks really good well done 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (bimbolabuari) [reply index.html]: Hi @Davidosky007, Yes, you are right! 👍 I actually planned to do this but I must have missed it as I did in other places, see the permalinks below:

>https://github.com/bimbolabuari/mv-portfolio/blob/eed2622751568e6f614e89b29e8203ca3ddb97f5/styles.css#L50

File: styles.css (hunk 1 above)
Comment (bimbolabuari) [reply styles.css:222]: Hi @Davidosky007, I think this requirement is done, 🙂  you can see that I used `display: grid` in the container that holds all the cards, see the code and quote from the requirement below

> Use CSS Grid in the main container that contains all of the cards and use Flexbox to layout an individual card.

>https://github.com/bimbolabuari/mv-portfolio/blob/eed2622751568e6f614e89b29e8203ca3ddb97f5/styles.css#L204-L206

**Comment by bimbolabuari** [comment]
Hi @Davidosky007, Thanks for your review 👍 🙂

File: styles.css (hunk 1 above)
Comment (Davidosky007) [reply styles.css:222]: Ok 👌 good job keep it up 👍

File: index.html (hunk 2 above)
Comment (Davidosky007) [reply index.html]: Nice 👍

**Review by henatan99 — APPROVED** [review]
Hi Abimbola, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bimbolabuari** [comment]
@henatan99, Thanks for approving my project 🚀

**Pull request merged**